tools/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -232,22 +236,45 @@ def create_statistics_html(self, output_dir):
                 checker = report['checkerName']
                 checker_statistics[checker] += 1
 
-        rows = []
+        checker_rows = []
         checker_statistics_content = ''
+        severity_statistics = defaultdict(int)
         for checker_name in sorted(checker_statistics):
             checker_statistics_content += '''
               <tr>
                 <td>{0}</td>
+                <td class="severity" severity="{1}">
+                  <i class="severity-{1}"></i>
+                </td>
+                <td>{2}</td>
+              </tr>
+            '''.format(checker_name, self._severity_map[checker_name].lower(),
+                       checker_statistics[checker_name])
+            checker_rows.append([checker_name,
+                                self._severity_map[checker_name],
+                                checker_statistics[checker_name]])
+            severity_statistics[self._severity_map[checker_name]] += 1
+
+        severity_rows = []
+        severity_statistics_content = ''
+        for severity in sorted(severity_statistics, key=severity_order):
+            num = severity_statistics[severity]
+            severity_statistics_content += '''
```

Comment:
this will not be a long string right? StringIO could be used to build larger strings.